axiom/tests/test_txn.c

```diff
@@ -8068,6 +8068,122 @@ static void test_segment_record_error(void) {
   nr_txn_destroy(&txn);
 }
 
+static void test_record_log_event(void) {
+  /* This is the plan for this test:
+      1. Setup state (create transaction with logging options)
+      2. Call nr_txn_record_log_event();
+      3. Check for state:
+        - log_event is added to transaction log events
+        - check if metrics are created
+  */
+}
```

Comment:
Is this just a placeholder for future work?

Reply:
@bduranleau-nr Correct.